### PR TITLE
fix: added contenttype to non-async flow and added release flow in ci

### PR DESCRIPTION
**Pull request by sidohaakma**

### CODE REVIEW

**Comment by sidohaakma** [comment]
[add debug info to all datashield service methods](https://trello.com/c/CqRiyfyh/374-add-debug-info-to-all-datashield-service-methods)

**Comment by sidohaakma** [comment]
[dsBaseClient errors](https://trello.com/c/hFXNujz5/373-dsbaseclient-errors)

**Comment by sidohaakma** [comment]
[added release flow to the armadillo](https://trello.com/c/aTenTyKP/325-added-release-flow-to-the-armadillo)

File: armadillo/src/main/java/org/molgenis/armadillo/ArmadilloUtils.java [hunk 1]
```diff
@@ -22,10 +27,13 @@ public static String serializeExpression(String cmd) {
     byte[] rawResult = new byte[0];
     if (result.isRaw()) {
       try {
+        LOGGER.trace(format("RAW result : [ %s ]", new String(result.asBytes(), UTF_8)));
         rawResult = result.asBytes();
       } catch (REXPMismatchException e) {
         throw new IllegalStateException(e);
       }
+    } else {
```
Comment (fdlk) [inline armadillo/src/main/java/org/molgenis/armadillo/ArmadilloUtils.java:35]: Let's just get really upset and throw exception if it's not a raw result. Fail fast, fail hard.

File: armadillo/src/main/java/org/molgenis/armadillo/DataController.java [hunk 2]
```diff
@@ -183,7 +183,10 @@ public void removeSymbol(@Valid @Pattern(regexp = SYMBOL_RE) @PathVariable Strin
   }
 
   @Operation(summary = "Execute expression")
-  @PostMapping(value = "/execute", consumes = TEXT_PLAIN_VALUE)
+  @PostMapping(
+      value = "/execute",
+      consumes = TEXT_PLAIN_VALUE,
+      produces = APPLICATION_OCTET_STREAM_VALUE)
```
Comment (fdlk) [inline armadillo/src/main/java/org/molgenis/armadillo/DataController.java:189]: nice catch!

File: armadillo/src/main/java/org/molgenis/armadillo/ArmadilloUtils.java [hunk 3]
```diff
@@ -22,10 +27,13 @@ public static String serializeExpression(String cmd) {
     byte[] rawResult = new byte[0];
     if (result.isRaw()) {
       try {
+        LOGGER.trace(format("RAW result : [ %s ]", new String(result.asBytes(), UTF_8)));
```
Comment (fdlk) [inline armadillo/src/main/java/org/molgenis/armadillo/ArmadilloUtils.java:30]: check LOGGER.isTraceEnabled() to save some time (sonar warning)

File: Jenkinsfile [hunk 4]
```diff
@@ -118,13 +118,46 @@ pipeline {
...
             }
         }
-    }
-    post {
-        success {
-            hubotSend(message: 'Build success', status: 'INFO', site: 'slack-pr-app-team')
-        }
-        failure {
-            hubotSend(message: 'Build failed', status: 'ERROR', site: 'slack-pr-app-team')
+        stage('Steps: [ x.x ]') {
```
Comment (fdlk) [inline Jenkinsfile:121]: Not completely sure about the x.x approach as opposed to only releasing from master. I would hope the service is simple enough that we can simply steam forward in a straight line ahead without the need for backports and stable branches.

Comment (fdlk) [reply Jenkinsfile:121]: But am happy to go ahead with this if it's easiest and/or you think it's the smart thing to do

**Pull request merged**